lib/sycamore/sycamore/data/table.py

```diff
@@ -421,7 +422,7 @@ def draw(self, target: U) -> U:
         return try_draw_boxes(target, self.cells, color_fn=lambda _: "red", text_fn=lambda _, i: None)
 
 
-class TableTree(apted.helpers.Tree):
+class TableTree:
```

Comment:
the inheritance is important here I thought

Reply:
I did try it and it still seems to work. I think duck typing is sufficient here, though if you can find a counter-example I am open to it.